template.yaml

```
@@ -517,6 +517,16 @@ Resources:
                   - !GetAtt TerraformAutoscalingGroupRole.Arn
             Resource:
               - !Sub ${TerraformStateBucket.Arn}/*
+          - Sid: AccessToBucketWithSpecificEndpoint
```

Minor - Can we rename this to?

```suggestion
          - Sid: DenyAccessToPublicEndpoint
```

---

Updated.